ui/perfherder/alerts/SelectAlertsDropdown.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DropdownItem from 'reactstrap/lib/DropdownItem';
+import DropdownMenu from 'reactstrap/lib/DropdownMenu';
+import DropdownToggle from 'reactstrap/lib/DropdownToggle';
+import UncontrolledDropdown from 'reactstrap/lib/UncontrolledDropdown';
+import Input from 'reactstrap/lib/Input';
+import Label from 'reactstrap/lib/Label';
+
+import { alertStatus as status } from '../constants';
+
+class SelectAlertsDropdown extends React.PureComponent {
+  render() {
+    const {
+      selectAlertsByStatus,
+      user,
+      allSelected,
+      alertSummary,
+    } = this.props;
+    const { untriaged, triaged, all, none } = status;
+
+    return (
+      <React.Fragment>
+        <Label check className="pl-1">
+          <Input
+            data-testid={`alert summary ${alertSummary.id.toString()} checkbox`}
+            aria-labelledby={`alert summary ${alertSummary.id.toString()} title`}
```

Comment:
What's the purpose of this `aria-labelledby`? I feel like it's not used properly.